### ISSUE TITLE
Errors 'No 'Access-Control-Allow-Origin' header is present on the requested resource.'

### ISSUE THREAD

**Issue by cb29** [appbaseio/reactivesearch#1413]
Hello,

I'm using ReactiveSearch (3.7.4) in a React project. 
On a page I have placed the ReactiveSearch search system : 

<ReactiveBase
 // ES URL : https://login:pwd@esurl:port.
url={urlWithAuthentication}
// ES index.
app='index'
 .....
>
<div><CustomComponentWithFilters /></div>
<div><CustomComponentWithReactiveList /></div>
...

But, when I open/load the page I get several errors in Chrome Dev Tools:

Access to XMLHttpRequest at 'https://es-url:port/index/_msearch?' from origin 'https://www.mysite.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

And, some elements are not rendered : filters, results.... If I reload the page the previous missed elements are visible but other elements are not present...

In my ES Cluster, I use the default CORS settings 

http.cors.enabled: true
http.cors.allow-credentials: true
http.cors.allow-origin: "*"
http.cors.allow-methods: OPTIONS, HEAD, GET, POST, PUT, DELETE
http.cors.allow-headers: X-Requested-With, X-Auth-Token, Content-Type, Content-Length, Authorization, Access-Control-Allow-Headers, Accept

Is the authentication correctly configured? Is there an other method to make the authentication to my Cluster? Is there a method to debug the connexion between ES and ReactiveSearch?

Thanks for your help.

**Comment by ShahAnuj2610** [comment]
@cb29 Thanks for reporting. It would be a great if you can share a reproducible code sandbox example.

**Comment by cb29** [comment]
Hi ShahAnuj2610 !
after testing, placing elements step by step....I see these errors appears with filters. If I haven't filters, I haven't errors. I will close this issue to re-open an issue related to the filters.

**Comment by dangreaves** [comment]
Hey @cb29 - we are experiencing the exact same issue, where intermittently some of the queries are hitting `No 'Access-Control-Allow-Origin' header is present on the requested resource.` yet the backend is configured correctly. After refreshing the page, the queries go through fine.

Did you manage to solve your issue? I notice you didn't open another issue regarding filters.

**Comment by EnzoVezzaro** [comment]
is this issue still opened?

**Comment by siddharthlatest** [comment]
@EnzoVezzaro This issue is closed. However, a CORS error implies either a backend configuration issue or a possible issue with a server crash (i.e. this is just masking the real error) if it's happening intermittently.
